
<launch>
	<arg name="veh"/>
	<arg name="node_name" default="dt_street_light_controller_node"/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />

	<group ns="$(arg veh)">   
		<node pkg="dt_street_light_controller" type="dt_street_light_controller_node.py" name="dt_street_light_controller_node" output="screen">
			<rosparam command="load" file="$(find dt_street_light_controller)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
		</node>
		
	</group>
</launch>
